iceoryx_posh/test/moduletests/test_popo_toml_gateway_config_parser.cpp

```diff
@@ -20,201 +20,528 @@
 using namespace ::testing;
 using ::testing::_;
 
-// ======================================== Helpers ======================================== //
+using namespace iox::config;
 
+// ======================================== Helpers ======================================== //
+const std::string TestFile = "gwconfig_test.tmp";
+#ifndef _WIN32
+const std::string TempPath = "/tmp";
+const std::string TestFilePath = TempPath + "/" + TestFile;
+#else
+const std::string TempPath = std::getenv("TEMP");
+const std::string TestFilePath = TempPath + "\\" + TestFile;
+#endif
 
 // ======================================== Fixture ======================================== //
 class TomlGatewayConfigParserTest : public Test
 {
   public:
     void SetUp(){};
-    void TearDown(){};
+    void TearDown()
+    {
+        if (std::remove(TestFilePath.c_str()) != 0)
+        {
+            std::cerr << "Failed to remove temporary file '" << TestFilePath
+                      << "'. You'll have to remove it by yourself.";
```

Comment:
Please add `std::endl` at the end of `std::cerr`

Reply:
Looks like it's done.